src/modules/reserve-overview/graphs/InterestRateModelGraph.tsx

```diff
@@ -428,6 +430,23 @@ export const InterestRateModelGraph = withTooltip<AreaProps, TooltipData>(
                   {tooltipData.utilization}%
                 </Typography>
               </Box>
+
+              <Box sx={{ display: 'flex', justifyContent: 'space-between', mb: 2 }}>
+                <Typography variant="main12" color="primary" sx={{ mr: 2 }}>
+                  <Trans>Additional borrows to reach targeted utilization</Trans>
+                </Typography>
+                <Typography variant="main12" color="primary">
+                  $
+                  {new Intl.NumberFormat('en-US', {
+                    minimumFractionDigits: 2,
+                    maximumFractionDigits: 2,
+                  }).format(
+                    (tooltipData.utilization / 100) * parseFloat(reserve.totalLiquidityUSD) -
+                      parseFloat(reserve.totalDebtUSD)
```

Comment:
I think instead of `utilization` you want to use `formattedOptimalUtilizationRate`.
Also, for V3, reserves have borrow cap, i think you should  check if `reserve.borrowCap` exists, and instead of `totalLiquidityUSD` you should use `borrowCap` if it does.
Should we change the text in the case of negative numbers? Or we can directly not show this, what do you think?